sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -279,7 +291,11 @@ export class StreamingReceiver extends MessageReceiver {
             bMessage.messageId,
             this.name
           );
-          this._onError!(err);
+          this._onError!(err, {
+            errorSource: "processMessageCallback",
```

Comment:
A few lines above we have a `if(!isAmqpError(err))` check, meaning we wont be calling the user's error callback for any error thrown from the user's message callback that may have originated by them doing any of the client operations like message settling or renewing lock on their own.

Reply:
This should be outside of the if-check. Users should be able to see all the errors, not just the non-amqp ones.

Reply:
Agreed - it's nice to review the old code sometimes :)

Reply:
I've removed the if-check in both session and non-session receivers. It is odd. :|